src/OpenTelemetry.Exporter.Zipkin/ZipkinExporter.cs

```diff
@@ -57,33 +57,30 @@ public ZipkinExporter(ZipkinExporterOptions options, HttpClient client = null)
         internal ZipkinEndpoint LocalEndpoint { get; }
 
         /// <inheritdoc/>
-        public override async Task<ExportResult> ExportAsync(IEnumerable<Activity> batchActivity, CancellationToken cancellationToken)
+        public override ExportResultSync Export(in Batch<Activity> batch)
         {
             try
             {
-                await this.SendBatchActivityAsync(batchActivity, cancellationToken).ConfigureAwait(false);
+                // take a snapshot of the batch
+                var activities = new List<Activity>();
+                foreach (var activity in batch)
+                {
+                    activities.Add(activity);
+                }
+
+                this.SendBatchActivityAsync(activities, CancellationToken.None).RunSynchronously();
 
-                return ExportResult.Success;
+                return ExportResultSync.Success;
             }
             catch (Exception ex)
             {
                 ZipkinExporterEventSource.Log.FailedExport(ex);
 
                 // TODO distinguish retryable exceptions
```

Comment:
Is this still a relevant TODO?

Reply:
dont think so. but not because of this PR.

Reply:
I guess it is still a TODO, the scheduler and flush (and potentially offline storage) part still needs to be improved.

Reply:
Yes to that part.

I meant that there is no distinct return type for Failed vs FailedButRetryable.  Spec was modified to have only 2 return types back to Processor- Success or Failure.

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#span-exporter